Libplanet/Blockchain/Renderers/Debug/RenderRecord.cs

```diff
@@ -9,7 +9,7 @@ namespace Libplanet.Blockchain.Renderers.Debug
     /// </summary>
     /// <typeparam name="T">An <see cref="IAction"/> type.  It should match to
     /// <see cref="Libplanet.Blockchain.BlockChain{T}"/>'s type parameter.</typeparam>
-    public abstract class RenderRecord<T>
+public abstract class RenderRecord<T>
```

Comment:
This change seems not related to this pull request 🤔 This change should be reverted.